README.md

```diff
@@ -111,10 +104,6 @@ To build GoldenDict with Visual Studio take one of next library packs and unpack
 [GoldenDict_libs_VS2015_x86_v4.7z](http://www.mediafire.com/file/0a7ygy9rn99oevm/GoldenDict_libs_VS2015_x86_v4.7z) - for MS Visual Studio 2015, 32 bit  
 [GoldenDict_libs_VS2015_x64_v4.7z](http://www.mediafire.com/file/yoy2q8af0s1467m/GoldenDict_libs_VS2015_x64_v4.7z) - for MS Visual Studio 2015, 64 bit  
 
-To create project files for Visual Studio you can pass `"-tp vc"` option to `qmake`.
```

Comment:
Why did you remove this line? Is creating a Visual Studio project like this no longer possible in Qt 5?

Reply:
possible ,but not needed.
visual studio can open goldendict.pro  directly .

Reply:
reverted

Reply:
Some developers may prefer generating native project files. Thanks for reverting.